docs/css/extra.css

```diff
@@ -1,6 +0,0 @@
-/*
```

Comment:
It was a hack for removing the first heading from the toc, otherwise it also appeared.